src/rt/cast_.d

```diff
@@ -84,37 +84,6 @@ int _d_isbaseof2(ClassInfo oc, ClassInfo c, ref size_t offset)
         if (oc.base is c)
             return true;
 
-      version(all)
-      {
-        // Bugzilla 2013: Until the ClassInfo.interfaces[] sturcture is fixed
-        // (it's generated by compiler), we should keep breadth-first search
-        // to avoid existing code breaking.
-
-        // Before the compiler fix:
-        //  When oc is a ClassInfo of interface type, iface.offset is always zero.
-        //  So the offset addition has no operation.
-        // After the compiler fix:
-        //  When oc is a ClassInfo of interface type, iface.offset may be nonzero.
-        foreach (iface; oc.interfaces)
-        {
-            if (iface.classinfo is c)
-            {
-                offset += iface.offset;
-                return true;
-            }
-        }
-
-        foreach (iface; oc.interfaces)
-        {
-            if (_d_isbaseof2(iface.classinfo, c, offset))
-            {
-                offset += iface.offset;
-                return true;
-            }
-        }
-      }
-      else
-      {
         // Bugzilla 2013: After the compiler is actually fixed, we can simply use
```

Comment:
I updated the comment.
A correct bugzilla number in comment is a good note to know why the code is formed as so, if the code is not yet mature enough.